contracts/mocks/tests/MockReserveInterestRateStrategy.sol

```diff
@@ -1,12 +1,12 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.10;
 
-import {IReserveInterestRateStrategy} from '../../interfaces/IReserveInterestRateStrategy.sol';
 import {IPoolAddressesProvider} from '../../interfaces/IPoolAddressesProvider.sol';
 import {WadRayMath} from '../../protocol/libraries/math/WadRayMath.sol';
 import {DataTypes} from '../../protocol/libraries/types/DataTypes.sol';
+import {IDefaultInterestRateStrategy} from "../../interfaces/IDefaultInterestRateStrategy.sol";
 
-contract MockReserveInterestRateStrategy is IReserveInterestRateStrategy {
+contract MockReserveInterestRateStrategy is IDefaultInterestRateStrategy {
```

Comment:
Would be nice to check if a `MockReserveInterestRateStrategy` that only implements `IReserveInterestRateStrategy` is enough to satisfy the test suite. if so, I would be in favor of pruning this mock contract.

Reply:
It is enough (tested it) but I think its ok to leave the full mock, as for what i have seen, all interest rate strategies use all the methods, so could be used for someone